pkg/get/tools.go

```diff
@@ -237,5 +237,36 @@ https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}
 `,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "digitalocean",
+			Repo:    "doctl",
+			Name:    "doctl",
+			Version: "1.46.0",
+			URLTemplate: `
+		{{$osStr := ""}}
+		{{ if HasPrefix .OS "ming" -}}
+		{{$osStr = "windows"}}
+		{{- else if eq .OS "linux" -}}
+		{{$osStr = "linux"}}
+		{{- else if eq .OS "darwin" -}}
+		{{$osStr = "darwin"}}
+		{{- end -}}
+			
+		{{$archStr := ""}}		
+		{{- if eq .Arch "x86_64" -}}
+		{{$archStr = "amd64"}}						
+		{{- end -}}		
+
+		{{$archiveStr := ""}}		
+		{{ if HasPrefix .OS "ming" -}}
+		{{$archiveStr = "zip"}}		
+		{{- else -}}
+		{{$archiveStr = "tar.gz"}}		
+		{{- end -}}
+
+		https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/{{.Name}}-{{.Version}}-{{$osStr}}-{{$archStr}}.{{$archiveStr}}`,
```

Comment:
as mentioned above, use the actual value of owner, repo etc here rather than passing them in each time

Reply:
And here. Thank you!